pkg/apis/integreatly/v1alpha1/rhmi_types.go

```diff
@@ -57,7 +57,7 @@ var (
 	// It is preferable to determine the version of a product dynamically (from a CR, or configmap, etc)
 	// Follow up Jira: https://issues.redhat.com/browse/INTLY-5946
 	VersionAMQOnline           ProductVersion = "1.4"
-	VersionApicurito           ProductVersion = "7.6"
+	VersionApicurito           ProductVersion = "7.7"
```

Comment:
@philbrookes What version are we putting next to apicurito here? The version of fuse online it gets distributed with?